packages/sdk/tasks/pauseMarketMinting.ts

```diff
@@ -0,0 +1,43 @@
+import { task, types } from "hardhat/config";
+import { CToken } from "../lib/contracts/typechain/CToken";
+import { Comptroller } from "../lib/contracts/typechain/Comptroller";
```

Comment:
these imports are unlikely to work -- in my experience, you can't do such imports in tasks. Did you test this locally?

Reply:
this worked, right?